### PR TITLE
added more badges with links

### LINKED ISSUE #314: add more badges to README.md file
### Can you please describe the problem to be solved?

add more badges to the top of readme.md file, since the badges will be colorful and more attractive at the top of README.md file than the text the users will like it and it will also help them to navigate to different sections/ webpages like PRs, issues, contributors, etc.

### Can you give some suggestions?
I will add some more badges so that the README.md file will look more attractive and awesome. And also I will align them all to the center.

### Screenshots (present)
I want to add more badges here

image

### PR DESCRIPTION
**Pull request by Umesh-01**
### What does this implement/fix? Explain your changes.
added more badges with links to the readme.md file, it will help the users to navigate to different sections/ webpages like PRs, issues, contributors, etc.

### Does this close any currently open issues?
This fixes #314 

### How did you test your changes?
Manually tested and verified the changes.

### Any relevant logs, error output, etc.?

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,33 @@
...
+
+<a href="https://github.com/topics/html5"><img alt="HTML5" src="https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white"/></a>
+<a href="https://github.com/topics/php"><img alt="PHP" src="https://img.shields.io/badge/php-%23777BB4.svg?&style=for-the-badge&logo=php&logoColor=white"/></a> 
+<a href="https://github.com/topics/javascript"><img alt="JavaScript" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/></a> 
+<a href="https://github.com/topics/python"><img alt="Python" src="https://img.shields.io/badge/python%20-%2314354C.svg?&style=for-the-badge&logo=python&logoColor=white"/></a> 
+<a href="https://github.com/topics/css3"><img alt="CSS3" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white"/></a>
+<a href="https://github.com/topics/bootstrap"><img alt="BootStrap" src="https://img.shields.io/badge/Bootstrap-563D7C?style=for-the-badge&logo=bootstrap&logoColor=white"/></a> 
+</div>
```
Comment (siv2r) [inline README.md:30]: Can you move these new changes to the top of the file?

**Review by siv2r — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Umesh-01) [reply README.md:30]: @siv2r did u mean before the logo or at the top (before the heading)??

Comment (siv2r) [reply README.md:23]: @Umesh-01 before the heading (line 1 of this page)

Comment (Umesh-01) [reply README.md:23]: done, check it now @siv2r

**Review by siv2r — CHANGES_REQUESTED** [review]

**Comment by Umesh-01** [comment]
@siv2r done done.. 👍

**Comment by siv2r** [comment]
Great work!!
Thanks for the contribution :)

**Comment by Umesh-01** [comment]
thank u @siv2r 🙂

**Pull request merged**